localtypings/pxtarget.d.ts

```diff
@@ -284,6 +284,7 @@ declare namespace pxt {
         organizationUrl?: string;
         organizationLogo?: string;
         organizationWideLogo?: string;
+        skillmapOrganizationLogo?: string;
```

Comment:
yeah I was having some trouble getting the two paths to line up nicely between skillmap and regular app. I'll work on it again

Reply:
you might need to do something different for localhost/if the apptheme isn't present (i think skillmap currently shows the generic makecode logo instead of the target logo on localserve), but in production the theme path should work

Reply:
oh hmmmmmmmm okay trust

Reply:
Seeing a broken logo in skillmap now. Related?
![image](https://user-images.githubusercontent.com/12176807/128412546-71899f7c-8084-40c1-a484-8e958dd8e9d6.png)

Reply:
yeah you might need to do an isLocal() check and switch the url based on whether it's localhost or not (not required for this pr imo!)